src/converters.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  EntityFromIntegration,
+  RelationshipFromIntegration
+} from "@jupiterone/jupiter-managed-integration-sdk";
+import { Agent, WazuhManager } from "./provider";
+
+export const ACCOUNT_ENTITY_TYPE = "provider_account";
+export const ACCOUNT_ENTITY_CLASS = "Account";
+
+export const WAZUH_MANAGER_ENTITY_TYPE = "provider_wazuhmanager";
+export const WAZUH_MANAGER_ENTITY_CLASS = "Wazuhmanager";
+
+export const AGENT_ENTITY_TYPE = "provider_agent";
+export const AGENT_ENTITY_CLASS = "Agent";
+
+export const ACCOUNT_WAZUH_MANAGER_RELATIONSHIP_TYPE =
+  "provider_account_wazuhmanager";
+export const ACCOUNT_WAZUH_MANAGER_RELATIONSHIP_CLASS = "HAS";
+
+export const WAZUH_MANAGER_AGENT_RELATIONSHIP_TYPE =
+  "provider_wazuhmanager_agent";
+export const WAZUH_MANAGER_AGENT_RELATIONSHIP_CLASS = "HAS";
+
+export interface WazuhManagerEntity
+  extends EntityFromIntegration,
+    WazuhManager {
+  managerId: string;
+}
+
+export interface AgentEntity extends EntityFromIntegration, Agent {
+  ownerId: string;
+}
+
+export function createWazuhManagerEntities(
+  data: WazuhManager
+): WazuhManagerEntity {
+  // Note the manager is also the account entity
+  return {
+    _key: `${WAZUH_MANAGER_ENTITY_TYPE}-${data.id}`,
+    _type: WAZUH_MANAGER_ENTITY_TYPE,
+    _class: [WAZUH_MANAGER_ENTITY_CLASS, ACCOUNT_ENTITY_CLASS],
```

Comment:
As mentioned, `Account` is sufficient. The other class does not fit the data model. Also, if we add another class, `WAZUH_MANAGER_ENTITY_CLASS` should become an `Array` of class values.

```suggestion
    _class: WAZUH_MANAGER_ENTITY_CLASS,
```